lib/tutorials/en_US/server-methods.md

```diff
@@ -104,9 +128,9 @@ server.method('add', add, {
 server.methods.add(5, 12);
```

Comment:
It's a little awkward because we're not in an `async` closure here, but this call method should probably `await`ed.

Reply:
I can just remove it as well...doesn't really add to the tutorial